builder.json

```diff
@@ -87,7 +87,23 @@
                     ],
                     "!test_steps": [
                         "aws-crt-java-test"
-                    ]
+                    ],
+                    "!run_tests": true,
+                    "architectures": {
+                        "armv7": {
+                            "_comment": [
+                                "Due to inability to differentiate at runtime armv6 vs armv7 in the jvm",
```

Comment:
A comment about why we can't differentiate at runtime would be greatly appreciated.

Reply:
well, I also don't know... Just copy/paste from the comments in our CI before.